### PR TITLE
fix #1 fix #6 feature #1 feature #2 feature #3 added with exception for different same names and symbol

### LINKED ISSUE #6: Draw Game
shows player 1 wins for DRAW match.

### PR DESCRIPTION
**Pull request by iosdev747**
fix #1 
fix #6 
resolved and added custom symbols with exceptions for same name and same symbol and on reclicking on same position i.e. occupied.

### CODE REVIEW

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 1]
```diff
@@ -96,14 +96,23 @@ private void check(int currentPlayer){
     
     private void cellClicked(MouseEvent evt){
         JLabel currentCell = (JLabel) evt.getComponent();
-        if(currentPlayer == 0){
+        String temp = currentCell.getText();
+        //System.out.println(temp + (temp == "."));
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java]: Remove unnecessary comments.

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 2]
```diff
@@ -96,14 +96,23 @@ private void check(int currentPlayer){
...
+        if(temp == "."){
+            if(currentPlayer == 0){
             currentCell.setText("O");
+            jLabel4.setText("Player: 2");
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java]: Use meaningful names.

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 3]
```diff
@@ -96,14 +96,23 @@ private void check(int currentPlayer){
...
+            else{
+                currentCell.setText("X");
+                jLabel4.setText("Player: 1");
+            }
+            currentPlayer = (currentPlayer + 1) % 2;
+            check( (currentPlayer + 1) % 2 );
         }
         else{
-            currentCell.setText("X");
+        JOptionPane.showMessageDialog(null, "Not an empty place!!!");
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java]: Properly indent the code

File: TicTacToe/src/tictactoe/GameWindow.java (hunk 1 above)
Comment (iosdev747) [reply TicTacToe/src/tictactoe/GameWindow.java]: done

**Review by iosdev747 — COMMENTED** [review]
adding another feature so closing PR :)

**Pull request closed without merging**